src/dataModels/JSON.js

```diff
@@ -607,42 +614,20 @@ var JSON = DataModel.extend('dataModels.JSON', {
 
     /**
      * @memberOf dataModels.JSON.prototype
-     * @param {number} y
+     * @param {number} r - Grid row coordinate.
      * @returns {object}
      */
-    getRow: function(y) {
-        var headerRowCount = this.grid.getHeaderRowCount();
-        var topTotals = this.getTopTotals();
-        var hasToptotals = !!topTotals.length;
-        if (y < headerRowCount && !hasToptotals) {
+    getRow: function(r) {
+        var headerRowCount = this.grid.getHeaderRowCount(),
+            topTotals = this.getTopTotals(),
+            hasToptotals = !!topTotals.length,
+            y = r - headerRowCount;
 
-            return topTotals[y - (headerRowCount - topTotals.length)];
+        if (r < headerRowCount && !hasToptotals) {
+            return topTotals[r - (headerRowCount - topTotals.length)];
         }
-        return this.dataSource.getRow(y - headerRowCount);
-    },
-
-    buildRow: function(y) {
-        return this.deprecated('buildRow', 'dataSource', '1.0.8');
-    },
-
-    /**
-     * @memberOf dataModels.JSON.prototype
-     * @param {number} y
-     * @returns {object}
-     */
-    getComputedRow: function(y) {
-        var rcf = this.getRowContextFunction([y]);
-        var fields = this.getFields();
-        var row = {};
-        for (var i = 0; i < fields.length; i++) {
-            var field = fields[i];
-            row[field] = rcf(field)[0];
-        }
-        return row;
-    },
 
-    getValueByField: function(fieldName, y) {
-        return this.deprecated('getValueByField', 'dataSource', '1.0.8');
```

Comment:
no, we're deleting it. you didn't see [my comment](https://github.com/openfin/fin-hypergrid/commit/391d6ea35e16a5279092de87a81abf257c38ba84)...